### ISSUE TITLE
Placeholders in `html_title` config option are not expanded

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1082]
The place holders in the `html_title` ([link](http://sphinx-doc.org/config.html#confval-html_title)) configuration option do not appear to get expanded.

I have attempted to change the HTML title to:

```
html_title = "<project> documentation"
```

but when it is rendered in the HTML I get:

```
<title>&lt;project&gt; documentation</title>
```

It looks like this is an issue that has been around for a while, but I couldn't find any tickets here for it.

See: http://osdir.com/ml/python-sphinx-devel/2009-02/msg00221.html for a work around in the mean time.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1082
- Originally reported by: Simon Holywell
- Originally created at: 2013-01-16T15:15:19.945

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-01-17 09:20:50+00:00_

> The place holders in the `html_title` ([link](http://sphinx-doc.org/config.html#confval-html_title)) configuration option do not appear to get expanded.

I can not find that functionality in current implementation.
[Georg Brandl](https://bitbucket.org/birkenfeld), please confirm this.

I think `placeholders are replaced` part of the document is not good.

> It defaults to '<project> v<revision> documentation', where the placeholders are replaced by the config values of the same name.

**Comment by shimizukawa** [comment]
_From Simon Holywell on 2013-01-18 13:35:19+00:00_

That makes sense. I hadn't thought of it as just a way of documenting the default title.

To me it implied that I could just override the default and amend it as I pleased using the `<placeholders>` shown.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-11 20:03:24+00:00_

Closes #1082: clarify that html_title default value does not really have placeholders

→ <<cset d38c5cac8d418f8f1a7fa2831e51c31618e6dae0>>
